### ISSUE TITLE
build scripts don't support [BUILD] and [NOBUILD]

### ISSUE THREAD

**Issue by dgobaud** [ueno-llc/react-native-starter#204]
Documentation https://github.com/ueno-llc/react-native-starter/blob/master/docs/PUBLISHING.md image shows scripts looking for [BUILD] and [NOBUILD] tags in commit messages but scripts don't seem to do that

**Comment by dgobaud** [comment]
I tried adding below to android/build.gradle but it doesn't seem to execute the script and I can't figure out why

```
task buildEnv(type: Exec) {
    workingDir '../'
    
    commandLine './scripts/build-env.sh'
}
```

**Comment by jeremybarbet** [comment]
Hey! Sorry for the big delay.

You are totally right, `[BUILD]` and `[NOBUILD]` don't work. There is a simple reason for that, they are not part of any scripts anymore. We were using bitrise before, and this documentation was targeting the bitrise app. Since then we swiched to Appcenter, and we kept the same pipeline image and forget about this. We could take a look to bring them back into the appcenter-scripts-*
